docs/changelog/93198.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 93198
+summary: Add new sparse scored terms mapped field
+area: "Search, Machine Learning"
```

Comment:
This defines which section of the release notes the feature is listed in, so I believe it has to list a single area.

Reply:
I will update the notes for sure to only say Search :). But wanted to ping both teams.